### PR TITLE
Show batch counter ui only for multibatch dispense

### PR DESCRIPTION
**Pull request by kokipsiho**

### CODE REVIEW

File: ui/src/app.js [hunk 1]
```diff
@@ -1206,10 +1206,18 @@ function fiatComplete (tx) {
...
   setState('fiat_complete')
 }
 
+function toggleBatchCounter(state) {
+  const opacity = state === 'show' ? 100 : 0
+  $('.batch').css('opacity', opacity)
+}
+
 function dispenseBatch (data) {
+  if (data.of > 1) toggleBatchCounter('show')
```
Comment (joshmh) [inline ui/src/app.js]: It's safer to hide here for batches of 1, otherwise we're assuming that fiatComplete has already run since the last toggle.

Comment (kokipsiho) [reply ui/src/app.js]: Makes sense

File: ui/src/app.js [hunk 2]
```diff
@@ -1206,10 +1206,18 @@ function fiatComplete (tx) {
...
   setState('fiat_complete')
 }
 
+function toggleBatchCounter(state) {
+  const opacity = state === 'show' ? 100 : 0
+  $('.batch').css('opacity', opacity)
+}
+
 function dispenseBatch (data) {
+  if (data.of === 1) toggleBatchCounter('hide')
```
Comment (joshmh) [inline ui/src/app.js]: I meant that you should handle both cases: hide if ``data.of === 1`` and show if ``data.of > 1``.

File: test/unit/test-optimize-batch.js [hunk 3]
```diff
@@ -35,3 +36,10 @@ test('Sum of optimized batches equals input', t => {
 
   t.is(_.sum(_.flatten(o)), 66)
 })
+
+test('Should return input array', t => {
+  const input = [7,15]
+  const o = optimizeDispenseWithoutLimit(input)
+  
+  t.deepEqual(o, input)
+})
```
Comment (RafaelTaranto) [inline test/unit/test-optimize-batch.js]: This test will fail after https://github.com/lamassu/lamassu-machine/pull/195 gets merged. It should return [[7,15]].

**Comment by RafaelTaranto** [comment]
@joshmh this PR can be closed without merging!

**Pull request closed without merging**